app/models/template.rb

```diff
@@ -88,9 +88,13 @@ def import_without_save(text, options = {})
     self.template = text
     @importing_metadata = self.class.parse_metadata(text)
     Foreman::Logging.logger('app').debug "setting attributes for #{name} with id: #{id || 'N/A'}"
-    self.snippet = !!@importing_metadata[:snippet]
+    self.snippet = !!@importing_metadata["snippet"]
```

Comment:
nitpick: given you decided to use string key, I'd prefer single quote strings as those are bit more efficient to evaluate:
```suggestion
    self.snippet = !!@importing_metadata['snippet']
```

Reply:
so I am not sure If I have decided that.
when I check the `@importing_metadata` output the keys appear as a string and not as a symbol. Is there a reason why?

Reply:
Because it is parsed as [`ActiveSupport::HashWithIndifferentAccess`](https://api.rubyonrails.org/classes/ActiveSupport/HashWithIndifferentAccess.html)